app/src/main/cpp/skyline/common.h

```diff
@@ -338,6 +339,18 @@ namespace skyline {
         constexpr std::size_t Hash(std::string_view view) {
             return frz::elsa<frz::string>{}(frz::string(view.data(), view.size()), 0);
         }
+
+        /**
+         * @brief Fills an array with random data from a Mersenne Twister pseudo-random generator
+         * @note Currently the generator is not seeded so produced values will be identical between two runs
+         */
+        template<typename T, typename S = u64> requires (std::is_integral_v<T> && std::is_integral_v<S>)
+        void GetRandomBytes(std::span<T> bytes, S seed = GetTimeTicks()) {
```

Comment:
Why is the span parameter called `bytes` when it's not a byte-span, I also disagree with the function naming as it's not really a getter as much as it is filling a `span` with random data. I'd recommend something like `FillRandomBytes` rather than `GetRandomBytes`.